backend/api/Controllers/RobotController.cs

```diff
@@ -569,10 +569,10 @@ public async Task<ActionResult> PauseMission([FromRoute] string robotId)
                 await errorHandlingService.HandleLosingConnectionToIsar(robot.Id);
                 return StatusCode(StatusCodes.Status502BadGateway, Message);
             }
-            catch (MissionException e)
+            catch (MissionPauseException e)
             {
                 logger.LogError(e, "Error while pausing ISAR mission");
-                return StatusCode(StatusCodes.Status502BadGateway, $"{e.Message}");
+                return StatusCode(StatusCodes.Status500InternalServerError, $"{e.Message}");
```

Comment:
Could we do a 400 error instead and say something generic like "Unable to pause mission"?